src/main/java/hudson/plugins/jira/selector/jobs/JobIssueSelector.java

```diff
@@ -0,0 +1,34 @@
+package hudson.plugins.jira.selector.jobs;
```

Comment:
Great changes, I like this now!
One thing only - as the "Job"s term are coming from Perforce and in Jenkins there is a similar term for a Job, I find this package name confusing. I propose to rename: hudson.plugins.jira.selector.jobs -> udson.plugins.jira.selector.perforce and move all classes there.
If you agree and this is done, I think we'll be good to merge!